build_for_ultimaker.sh

```diff
@@ -53,7 +53,7 @@ INITRAMFS_IMG="${KERNEL_BUILD}/initramfs.cpio${INITRAMFS_COMPRESSION}"
 GEN_INIT_CPIO="${KERNEL_BUILD}/usr/gen_init_cpio"
 GEN_INITRAMFS_LIST="${KERNEL}/scripts/gen_initramfs_list.sh"
 
-BB_PKG="http://ftp.nl.debian.org/debian/pool/main/b/busybox/busybox-static_1.22.0-19+b3_armhf.deb"
+BB_PKG="http://dl-cdn.alpinelinux.org/alpine/latest-stable/main/x86_64/busybox-static-1.28.4-r1.apk"
```

Comment:
We still need to create our own debian busybox package

Reply:
nah, we need to create our own busybox binary, it doesn't matter if its debian or apk or whatever, as it will only go into the initramfs. As such, hopefully, we can use the toolbox as is, to replace our initrd completely.